### ISSUE TITLE
Using keySeparator: false in the config breaks things

### ISSUE THREAD

**Issue by coyotte508** [i18next/i18next-parser#81]
Version: **1.0.0-beta-6**

I'm using `"keySeparator": false` so I can use strrings rather than key: `t('Hello! How are you?')` instead of `t('welcome.message')`.

The reason is so that this can be extracted: `t('Logging in...')`. With `keySeparator` set to false, the dots should not cause problems.

However doing that causes a bunch of weird files to popup: locales/en/translationfalseKey1.json, locales/en/translationfalseKey2.json, ... One different file per language and per individual key.

Here is my JSON:

```json
{
  "keySeparator": false,
  "namespaceSeparator": false,
  "lexers": {
    "js": ["JavascriptLexer", "JsxLexer"]
  },
  "sort": true,
  "locales": ["fr","en"]
}
```

Here is how I call with the cli: `i18next 'app/**/*.js' -c locales/i18next-parser.config.json -o .`

Current workaround: use something like `"__NO_KEY_SEPARATOR__"` instead of `false`.

**Comment by karellm** [comment]
@coyotte508 Thanks for the ticket. I will add support for `keySeparator: false` next time I fix bugs and prior releasing `1.0.0`. As you mentioned, right now the workaround is to set it to something that is unlikely to be used in keys (like `__NO_KEY_SEPARATOR__`).

**Comment by karellm** [comment]
I've just published `1.0.0-beta7` that address that issue.

**Comment by coyotte508** [comment]
Thank you!
